### PR TITLE
docs(ee) Clarify EE getting started docs and add screencasts

### PR DESCRIPTION
**Pull request by coopr**
https://github.com/Kong/getkong.org/blob/master/CONTRIBUTING.md

### Summary

Addition of Admin GUI instructions and screencasts to EE documentation

### Full changelog

* Clarified EE getting started docs
* Added screencast videos

### Checklist:

- [x] Spellchecked my documentation
- N/A [ ] Documentation 
- [x] Ready to be merged

### CODE REVIEW

**Comment by coopr** [comment]
Your review and merge is encouraged!

File: app/docs/enterprise/0.31-x/getting-started/adding-your-api.md [hunk 1]
```diff
@@ -10,13 +10,16 @@ step to having Kong EE manage your API. For purposes of this Getting Started
 guide, we suggest adding the [Mockbin API][mockbin] to Kong, as Mockbin is
 helpful for learning how Kong EE proxies your API requests.
 
-Kong exposes a [RESTful Admin API][API] on port `:8001` for managing the
-configuration of your Kong instance or cluster.
+Kong exposes a [RESTful Admin API][API] on ports `:8001` and `:8444` and an
+Admin GUI on ports `:8002` and `:8445` for managing the
+configuration of your Kong instance or cluster. The Admin GUI simply makes
```
Comment (hutchic) [inline app/docs/enterprise/0.31-x/getting-started/adding-your-api.md]: nitpick: avoid `simply`

File: app/docs/enterprise/0.31-x/getting-started/adding-your-api.md [hunk 2]
```diff
@@ -10,13 +10,16 @@ step to having Kong EE manage your API. For purposes of this Getting Started
...
+requests to the Admin API, so you can use either interface for configuring
+and managing Kong EE.
 
-### 1. Add your API using the Admin API
+## 1. Add your API using the Admin API or GUI
 
-Issue the following cURL request to add your first API ([Mockbin][mockbin])
-to Kong:
+If you'd like to use the Admin API, issue the following cURL request to add
```
Comment (hutchic) [inline app/docs/enterprise/0.31-x/getting-started/adding-your-api.md]: capitalize the c in CURL

Comment (coopr) [reply app/docs/enterprise/0.31-x/getting-started/adding-your-api.md]: Wikipedia suggests not https://en.wikipedia.org/wiki/CURL

Comment (hutchic) [reply app/docs/enterprise/0.31-x/getting-started/adding-your-api.md]: huh TIL

> The name originally stood for "see URL"

**Comment by coopr** [comment]
Yes @hutchic the video files should be versioned (to describe what version of Kong they are documenting) - I've made that change.

File: app/docs/enterprise/0.31-x/getting-started/quickstart.md [hunk 3]
```diff
@@ -45,8 +48,10 @@ By default Kong listens on the following ports:
 - `:8443` on which Kong listens for incoming HTTPS traffic. This port has a
   similar behavior as the `:8000` port, except that it expects HTTPS
   traffic only. This port can be disabled via the configuration file.
-- `:8001` on which the [Admin API][API] used to configure Kong listens.
+- `:8001` on which the [Admin API][API] listens.
 - `:8444` on which the [Admin API][API] listens for HTTPS traffic.
+- `:8002` on which the [Admin GUI][GUI] listens.
+- `:8445` on which the [Admin GUI][GUI] listens for HTTPS traffic.
```
Comment (darrenjennings) [inline app/docs/enterprise/0.31-x/getting-started/quickstart.md]: Should we add another point which is "If you enable the Kong Developer Portal, `portal=on`, then Kong will listen to :8003 and :8004. I could see both arguments a) this is a quickstart, don't bog down tutorial with developer portal, or b) give higher visibility to dev portal

Comment (coopr) [reply app/docs/enterprise/0.31-x/getting-started/quickstart.md]: Done, please review @darrenjennings

File: app/docs/enterprise/0.31-x/getting-started/quickstart.md [hunk 4]
```diff
@@ -52,6 +52,9 @@ By default Kong listens on the following ports:
 - `:8444` on which the [Admin API][API] listens for HTTPS traffic.
 - `:8002` on which the [Admin GUI][GUI] listens.
 - `:8445` on which the [Admin GUI][GUI] listens for HTTPS traffic.
+- `:8003` on which the [Admin GUI][GUI] listens for HTTP [Kong Dev Portal][dev-portal] traffic - if the Dev Portal is enabled.
```
Comment (darrenjennings) [inline app/docs/enterprise/0.31-x/getting-started/quickstart.md]: "on which ~Admin GUI~ Kong listens for HTTP..."

**Pull request merged**